### DISCUSSION TITLE
How to shutdown Longhorn on a Rancher cluster

### DISCUSSION THREAD

**Discussion by hamadodene** [longhorn/longhorn#4240] [Q&A]
We are planning maintenance for our Rancher cluster. We have several worker nodes and use longhorn as persistent storage.
What is the correct way to turn off the longhorn cluster and turn it back on at the end of the maintenance?

**Comment by mantissahz** [comment]
shutdown procedure for longhorn system: Scale down the workload and **make sure that all volumes are detached** and then turn off the cluster.

**Reply by innobead** [reply]
@mantissahz Do we have any documentation for this? If not, suggest updating the doc for the user reference. WDYT?

**Reply by mantissahz** [reply]
No, We don't have any document about shutdown procedure.
It's good to update the document to talk about this part.
Let me find somewhere to mention about shutdown procedure.

**Reply by hamadodene** [reply]
Thanks @mantissahz 
I also agree that it would be great to update the doc with this procedure
